pkg/infra/README.md

```diff
@@ -3,68 +3,68 @@
 根据Clusterfile里的定义去申请IaaS资源，保障IaaS资源与Clusterfile中的定义保持终态一致
 
 ```yaml
-apiVersion: xxx 
+apiVersion: xxx
 kind: Infra
 metadata:
-  name: alicloud-infra-demo
+  name: aws-infra-demo
+  annotations:
+    sealos.io/VpcID: v-xxxxx
+    sealos.io/RouteID: v-xxxxx
+    sealos.io/IgwID: v-xxxxxx
+    sealos.io/EgwID: v-xxxxxx
+    sealos.io/SubnetID: v-xxxxx
+    sealos.io/SecurityGroupID: v-xxxxx
+    sealos.io/EIPID: v-xxxxx
 spec:
   provider: AliyunProvider
   credential:
     accessKey: xxx
     accessSecret: xxx
   cluster:
-    isSeize: true
     regionIds: [cn-hangzhou, cn-shanghai]
-    zoneIds: [cn-hangzhou-a, cn-hangzhou-b] # If the value is empty, a random value will be used.
-    annotations: #set default value,if exist not delete resource
-        sealos.io/VpcID: v-xxxxx
-        sealos.io/VSwitchID: v-xxxxx
-        sealos.io/SecurityGroupID: v-xxxxx
-    accessChannels: # If all access channels are the same, it will be filled.
+    zoneIds: [cn-hangzhou-a, cn-hangzhou-b]
+    accessChannels:
       ssh:
-        passwd: xxx #If the passwd is empty, a random password will be generated.
+        username: ubuntu
+        # ENUM: passfile/pkfile
```

Comment:
Is the structure of direct reuse cluster.SSH a little bad?
```
type SSH struct {
	User     string `json:"user,omitempty"`
	Passwd   string `json:"passwd,omitempty"`
	Pk       string `json:"pk,omitempty"`
	PkPasswd string `json:"pkPasswd,omitempty"`
	Port     uint16 `json:"port,omitempty"`
}
```

Reply:
agree